### PR TITLE
Cumulative update after League Meet 1

### PR DESCRIPTION
**Pull request by kawaiiPlatypus**
This is a TENTATIVE pr, as we may have some things wrong in the commit

### CODE REVIEW

**Comment by kawaiiPlatypus** [comment]
This is a late night commit, I may have messed something up. Please check our quick coding and possibly loose logic

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriverControl.java [hunk 1]
```diff
@@ -72,16 +72,18 @@ public Gamepad gamepad2() {
...
-            if (gamepad2.dpad_left)
-                firmware.driveLinear(firmware.DRIVE_NORMAL() * dpadYAxisSign, 0.0);
-            else if (gamepad2.dpad_right)
-                firmware.driveLinear(0.0, firmware.DRIVE_NORMAL() * dpadYAxisSign);
-
-            if (gamepad1.dpad_down)
-                firmware.flipperLinear(1.0);
-            else if (gamepad1.dpad_up)
-                firmware.flipperLinear(-1.0);
+            /**
```
Comment (DarkLordZach) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriverControl.java]: Commenting!

**Comment by kawaiiPlatypus** [comment]
I DELETED THAT COME ONE

**Comment by DarkLordZach** [comment]
FIX ME!

**Comment by DarkLordZach** [comment]
PLEASE!

**Comment by kawaiiPlatypus** [comment]
Fixed in #50

**Pull request closed without merging**